source/scenario/Side.gd

```diff
@@ -1,6 +1,9 @@
 extends Node
 class_name Side
 
+signal side_add_unit
```

Comment:
signals use past tense. Also, since this signal belongs to Side, the side_ prefix is unnecessary.
rename this to `unit_added`